### PR TITLE
C#: Adding Membership.GeneratePassword() as a bad source of random data

### PR DESCRIPTION
**Pull request by raulgarciamsft**
Adding `Membership.GeneratePassword()` as a bad source of random data because of the bias caused by `% 87` operation against bytes of the randomly generated data.

### CODE REVIEW

**Comment by raulgarciamsft** [comment]
QQ. Where can I find information about the requirements for the `check-change-note` check? Thanks

**Comment by smowton** [comment]
Add a `.md` file like the ones in https://github.com/github/codeql/tree/main/csharp/change-notes

File: csharp/ql/src/Security Features/InsecureRandomness.cs [hunk 1]
```diff
@@ -15,7 +15,7 @@ string GeneratePassword()
         password = "mypassword" + BitConverter.ToInt32(randomBytes);
     }
 
-    // GOOD: Password is generated using a cryptographically secure RNG
+    // BAD: Membership.GeneratePassword is generates a password with a bias
```
Comment (intrigus-lgtm) [inline csharp/ql/src/Security Features/InsecureRandomness.cs:18]:
```suggestion
    // BAD: Membership.GeneratePassword generates a password with a bias
```

File: csharp/ql/test/query-tests/Security Features/CWE-338/InsecureRandomness.cs [hunk 2]
```diff
@@ -73,4 +73,24 @@ public static string InsecureRandomStringFromIndexer(int length)
         }
         return result;
     }
+
+    public static string BiasPasswordGeneration()
+    {
+        // BAD: Membership.GeneratePassword is generates a password with a bias
```
Comment (intrigus-lgtm) [inline csharp/ql/test/query-tests/Security Features/CWE-338/InsecureRandomness.cs:79]:
```suggestion
        // BAD: Membership.GeneratePassword generates a password with a bias
```

File: csharp/ql/src/Security Features/InsecureRandomness.cs (hunk 1 above)
Comment (raulgarciamsft) [reply csharp/ql/src/Security Features/InsecureRandomness.cs:18]: Thanks for the correction

Comment (intrigus-lgtm) [reply csharp/ql/src/Security Features/InsecureRandomness.cs:18]: Looks like you missed this one?

**Review by tamasvajk — APPROVED** [review]
@raulgarciamsft Thank you for this contribution. It LGTM, I only pushed some cosmetic changes.

**Pull request merged**